README.md

```diff
@@ -127,6 +129,11 @@ Gemfile:
 
 8) Remove your module from being managed via `modulesync` and you can start making use of `pdksync` going forward! Yay no more manual PR creation.
 
+### Compatibility
+----------
+
+Currently this tool has been ran and tested on OSX and linux. **It does not run on Windows.**
```

Comment:
Slight edit suggestion:
"This tool has been developed and tested on OSX and linux. **It currently does not run on Windows.**"